File: Java/TREE/Size of Binary Tree.java
//https://www.geeksforgeeks.org/problems/size-of-binary-tree/1
class Solution {
    public static int getSize(Node node) {
        // code here
        if(node==null){
            return 0;
        }
        int leftSize=getSize(node.left);
        int rightSize=getSize(node.right);
        return leftSize+rightSize+1;
    }
}

File: Python/TREE/Size of Binary Tree.py
//https://www.geeksforgeeks.org/problems/size-of-binary-tree/1
class Solution:
    def getSize(self, node : Optional['Node']) -> int:
        # code here
        if node is None:
            return 0
        left_size = self.getSize(node.left)
        right_size = self.getSize(node.right)
        return left_size + right_size + 1
